code/API_definitions/geofencing-subscriptions.yaml

```diff
@@ -409,8 +406,11 @@ components:
         subscriptionExpireTime:
           type: string
           format: date-time
-          example: "2023-01-17T13:18:23.682Z"
-          description: The subscription expiration time (in date-time format) requested by the API consumer.
+          description: |
```

Comment:
It seems that this description is only for the property expiresAt: https://github.com/camaraproject/Commonalities/blob/main/artifacts/camara-cloudevents/event-subscription-template.yaml#L510